.circleci/config.yml

```diff
@@ -23,6 +23,26 @@ jobs:
           command: |
             export commit_hash=`git rev-parse --short HEAD` && echo $commit_hash
             hokusai test
+  validate_staging_schema:
+    docker:
+      - image: circleci/node:10-stretch-browsers
+    parallelism: 1
```

Comment:
Is this needed? Taking a look at the [Circle docs](https://circleci.com/docs/2.0/parallelism-faster-jobs/#specifying-a-jobs-parallelism-level) it seems like default is 1 which implies no-parallelism?

> To run a job’s steps in parallel, set the parallelism key to a value greater than 1.